cli/serve/serve.go

```diff
@@ -112,23 +128,13 @@ var v1Endpoints = map[string]func() (http.Handler, error){
 		}
 		return apiocsp.NewHandler(ocspSigner), nil
 	},
-}
 
-var staticEndpoints = map[string]func() (http.Handler, error){
+	"/assets/": func() (http.Handler, error) {
+		return http.FileServer(rice.MustFindBox("static").HTTPBox()), nil
+	},
+
 	"/": func() (http.Handler, error) {
-		//the default rice order doesn't include working directory
-		order := []rice.LocateMethod{
-			rice.LocateEmbedded,
-			rice.LocateAppended,
-			rice.LocateFS,
-			rice.LocateWorkingDirectory,
-		}
-		c := &rice.Config{LocateOrder: order}
-		box, err := c.FindBox(staticDir)
-		if err != nil {
-			return nil, err
-		}
-		return http.FileServer(box.HTTPBox()), nil
+		return &box{rice.MustFindBox("static")}, nil
```

Comment:
This is breaking this test: https://github.com/cloudflare/cfssl/blob/serve-ui-updates/cli/serve/serve_test.go#L41